charts/opentelemetry-demo/README.md

```diff
@@ -27,6 +27,13 @@ helm install my-otel-demo open-telemetry/opentelemetry-demo
 
 ## Upgrading Chart
 
+### To 0.18
```

Comment:
How do you feel about moving this to an `UPGRADING.md` file?  This is what the other charts use.

Reply:
done